iacr/2021/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: IACR Crypto 2021
```

Comment:
I would prefer to not have nest folders for one-off talks. How many transcripts do we expect for IACR. The more nesting, the more clicks for the user.

Reply:
Done

Reply:
There will likely be more IACR transcripts. So keeping the IACR directory but got rid of the 2021 subdirectory